src/Model/Message.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Valantic\PimcoreFormsBundle\Model;
+
+use InvalidArgumentException;
+use Valantic\PimcoreFormsBundle\Http\ApiResponse;
+
+class Message extends AbstractMessage
+{
+    protected string $type;
+    protected string $message;
+    protected bool $expire;
+    protected int $delay;
+    protected array $source;
+    protected string $field;
+
+    public function setType(string $type): self
+    {
+        if (!in_array($type, ApiResponse::MESSAGE_TYPES, true)) {
```

Comment:
The `ApiResponse::*` constants could be refactored into e.g. `MessageConstants` as the current location of `ApiResponse` might be confusing. What do you think?

Reply:
I had it like this first and then changed it to use ApiResponse. Yeah you are right on second thought it makes sense to add `MessageConstants`